consensus/istanbul/core/core.go

```diff
@@ -377,7 +378,9 @@ func (c *core) commit() error {
 			c.waitForDesiredRound(nextRound)
 			return nil
 		}
-		if err := c.backend.Commit(proposal, aggregatedSeal, aggregatedEpochValidatorSetSeal); err != nil {
+
+		result, _ := c.current.GetStateProcessResult(proposal.Hash())
```

Comment:
why are we ignoreing the error?

Reply:
That's a `isCached` bool, not an error.
Since in the next line the `result` is going to be passed to `c.backend.Commit` anyway, and if the `result` is `nil`, we'll ignore it.